src/main/java/frc/robot/Constants.java

```diff
@@ -102,6 +102,7 @@ public final class Constants {
       public static final double ARM_ENCODER_COUNT_ERROR = 1.0;
       public static final double MAX_ARM_VELOCITY = 0.5;
       public static final double MAX_ARM_ACCELERATION = MAX_ARM_VELOCITY * 2;
+      public static final double ARM_ENCODER_COUNT_MAX_DIFF = -0.1;
```

Comment:
.1 is a pretty big difference , it’s bigger than the difference between mid and high.

also it looks like negative is the up direction based on the other constants